### ISSUE TITLE
Build CI system for Arm64 images

### ISSUE THREAD

**Issue by tenzen-y** [kubeflow/katib#1941]
/kind feature

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]
As discussed in https://github.com/kubeflow/katib/pull/1935#issuecomment-1221458383, we are currently working on #1900 to support arm64 in all container images, although we don't have any CI systems to verify operations on the arm64 machine.

So, we need to build a CI system to verify operations on the arm64 machine using GitHub self-hosted runner with [AWS Graviton2 EC2 instance](https://aws.amazon.com/ec2/graviton/).

/cc @johnugeorge @anencore94 

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

---

Love this feature? Give it a 👍 We prioritize the features with the most 👍

**Comment by midhun1998** [comment]
@tenzen-y Do you own this issue or are you looking for any help on this? 🙂 I'm willing to help.

**Comment by tenzen-y** [comment]
Thanks for your comments. @midhun1998 
I don't hold this issue. Although, we can not go ahead with this issue since we have not prepared an AWS account for WG automl yet.

**Comment by midhun1998** [comment]
Got it. Thanks for letting me know @tenzen-y . Let me know if you need help.

**Comment by djmcgreal-cc** [comment]
Hi. @andreyvelich . It looks like betas are being released with arm64 but releases aren't? Katib is now the only image in our platform that doesn't have an Arm64 image and we'd like to get the cost savings :)

Thanks, Dan.

**Comment by tenzen-y** [comment]
> Hi. @andreyvelich . It looks like betas are being released with arm64 but releases aren't?

You're right. We're working on it in https://github.com/kubeflow/katib/issues/2049.
